cmd/utils/flags.go

```diff
@@ -875,6 +875,56 @@ var (
 		Usage: "Default minimum difference between two consecutive block's timestamps in seconds",
 		Value: eth.DefaultConfig.Istanbul.BlockPeriod,
 	}
+
+	// Quorum Private Transaction Manager connection options
+	QuorumPTMUnixSocketFlag = DirectoryFlag{
+		Name:  "ptm.socket",
```

Comment:
I left the naming consistent with the toml config file (which has the name unchanged for backwards compatibility), in order to prevent confusion. However, if you think that's not important then I can change it.